notifications/README.md

```diff
@@ -0,0 +1,58 @@
+# Notifications
+
+## Notification Object
+
+```
+{
+  id: string,
+  username: string,
+  message: string,
+  createOn: unix-timestamp,
+  url: string,
+  type: string
+}
+```
+
+
+## **Requests**
+
+|               Route                |           Description           |
+| :--------------------------------: | :-----------------------------: |
+|      [GET /notifications](#get-notifications)      | Returns notifications of the logged in user |
+
+## **GET /notifications**
+
+Returns all notifications for a logged in user
+
+- **Params**  
+  None
+- **Query**  
+  page=[integer], n=[integer]
+- **Body**  
+  None
+- **Headers**  
+  Content-Type: application/json
+- **Cookie**  
+  rds-session: `<JWT>`
+- **Success Response:**
+- **Code:** 200
+  - **Content:**
+
+```
+{
+  message: 'Notifications returned successfully!',
+  notifications: [
```

Comment:
Can we call this `data` instead?
Otherwise, we are doing GET /notifications and then having to read a `.notifications` inside the response.